drizzlepac/haputils/catalog_utils.py

```diff
@@ -136,16 +136,43 @@ def compute_background(self, box_size, win_size,
         rms_estimator : subroutine
             RMS estimation algorithm
 
+        simple_bkg : bool, optional
+            Forces use of the sigma_clipped_stats algorithm
+
+        bkg_skew_threshold : float, optional
+            Discriminator on the skewness computation - below this limit the Background2D algorithm
+            will be computed for potential use for the background determination, otherwise
+            the sigma_clipped_stats algorithm is used.
+
+        negative_percent : float, optional
+            Discriminator on the background-subtracted image.  The percentage of negative values in the
+            background-subtracted image is determined - below this limit the Background2D algorithm stays in play,
+            otherwise the sigma_clipped_stats algorithm is used.
+
+        negative_threshold : float, optional
+            Value below which negative values are counted in the computation of the negative_percent
+
+        nsigma_clip : float, optional
+            Parameter for the sigma_clipped_stats algorithm - number of standard deviations to use for both
+            the lower and upper clipping limit.
+
+        maxiters : float, optional
+            Parameter for the sigma_clipped_stats algorithm - number of sigma-clipping iterations to perform
+
         Returns
         -------
-        bkg_background_ra : 2D ndarray
+        self.bkg_background_ra : 2D ndarray
```

Comment:
This doesn't make sense to me.  This implies that this method provides these updated values by explicitly passing that value back through the use of the 'return' command, yet, they are already attributes of the class and the code does not use the 'return' command.   It appears that this is meant more for documentation as to what attributes get updated with new values, but the 'Returns' section implies something very different.  Something different should be done here to avoid this confusion.